proto/src/determined/tensorboard/v1/tensorboard.proto

```diff
@@ -43,4 +44,8 @@ message Tensorboard {
   string service_address = 11;
   // The name of the resource pool the command was created in
   string resource_pool = 12;
+  // The config;
+  google.protobuf.Struct config = 13;
```

Comment:
we've kept config out of the list endpoints for experiments as it could end up accountable for the majority of the response size and instead only provide it for when a specific experiment is requested. I realize we probably won't accumulate as much commands as we would experiments but wdy think about keeping consistent with that approach?

```
// Response to GetExperimentRequest.
message GetExperimentResponse {
  option (grpc.gateway.protoc_gen_swagger.options.openapiv2_schema) = {
    json_schema: { required: [ "experiment", "config" ] }
  };
  // The requested experiment.
  determined.experiment.v1.Experiment experiment = 1;
  // The experiment config.
  google.protobuf.Struct config = 2;
}
```

also reminder to update the `required` annotation if config is always expected to be there